experimental/dds/tree2/src/shared-tree-core/sharedTreeCore.ts

```diff
@@ -260,15 +260,16 @@ export class SharedTreeCore<TEditor extends ChangeFamilyEditor, TChange> extends
 			commit,
 			sessionId: this.editManager.localSessionId,
 		});
-		this.submitLocalMessage(message);
+		this.submitLocalMessage(this.serializer.encode(message, this.handle));
```

Comment:
This is interesting, I wonder what GC implications this has. @markfields @agarwal-navin, this is how SharedTree is doing their handle work